tuf/client/updater.py

```diff
@@ -1142,12 +1142,10 @@ def neither_403_nor_404(mirror_error):
           # Otherwise, reraise the error, because it is not a simple HTTP
           # error.
           if neither_403_nor_404(mirror_error):
-            logger.exception('Misc error for root version '+str(next_version))
+            logger.info('Misc error for root version '+str(next_version) + str(exception))
```

Comment:
Nit: missing space between those two `'Misc error for root version '+str(next_version)`